### DISCUSSION TITLE
GitHub Student Pack Approved, but Benefits Not Received Over 12 days  and Redirect Issue

### DISCUSSION THREAD

**Discussion by abdelrahmanfm** [community/community#145108] [GitHub Education]
### Hi everyone,

Hello GitHub Education Team,

I hope this message finds you well. I am writing to address an issue with my GitHub Student Developer Pack application. Although my request was approved over 12 days ago, I have yet to receive any of the benefits or confirmation that the pack has been activated. Additionally, I am encountering a technical issue that makes it difficult to check my application details.

Here are the key points of my situation:

Approval Notification: My application for the GitHub Student Developer Pack was approved more than 12 days ago.
Benefits Not Activated: Despite the approval, I do not have access to any benefits associated with the pack.
Redirect Issue: Whenever I attempt to check my application details, I am automatically redirected to the GitHub pricing page at https://github.com/pricing, which prevents me from reviewing the status of my application.
I would appreciate your guidance on the following:

Is there a known delay in activating benefits after approval?
How can I resolve the issue of being redirected to the pricing page instead of my application details?
Are there additional steps I need to take to ensure proper activation of the Student Developer Pack?
As a student, I am eager to utilize the resources and tools provided by the GitHub Student Developer Pack to advance my education and projects. Your assistance in resolving this matter would mean a lot.

Thank you for your time and support. I look forward to hearing from you soon.

**Comment by davevad93** [comment]
Hi @abdelrahmanfm ,

The delay is normal and you don't have to take any further action, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

About the redirection to the [pricing page](https://github.com/pricing), it's a problem that sometimes affects other users too, I don't know why it happens, try contacting support by opening a ticket [here](https://support.github.com/contact/account), select `Other` and explain your issue.

However, even if you can't access your [application page](https://education.github.com/discount_requests/application), another way to know when benefits are available, is when a `PRO` badge shows up on your profile, that means you have GitHub Pro enabled therefore your benefits are finally available.

<img width="340" alt="0" src="https://github.com/user-attachments/assets/c40f72ea-826d-4a44-94ec-41a9c977a2cb">
